src/video_core/rasterizer_cache.h

```diff
@@ -89,48 +108,60 @@ class RasterizerCache : NonCopyable {
 
     /// Invalidates everything in the cache
     void InvalidateAll() {
+        std::lock_guard<std::recursive_mutex> lock{mutex};
+
         while (interval_cache.begin() != interval_cache.end()) {
             Unregister(*interval_cache.begin()->second.begin());
         }
     }
 
 protected:
-    /// Tries to get an object from the cache with the specified address
-    T TryGet(VAddr addr) const {
+    /// Tries to get an object from the cache with the specified cache address
+    T TryGet(CacheAddr addr) const {
         const auto iter = map_cache.find(addr);
         if (iter != map_cache.end())
             return iter->second;
         return nullptr;
     }
 
+    T TryGet(void* addr) const {
```

Comment:
This can be made:

```suggestion
    T TryGet(const void* addr) const {
```

if `ToCacheAddr`'s parameter is made const.